### PR TITLE
Project index2

### PR DESCRIPTION
**Pull request by amd711**
* Authentication and login 
related to #3 
* Project Index 
related to #5 
* search
related to #11

### CODE REVIEW

File: public/css/login.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    font-family: Roboto !important;
+}
+.login-card input[type=submit] {
+  width: 100%;
+  display: block;
+  margin-bottom: 10px;
+  position: relative;
+}
+
+.login-card input[type=email], input[type=password] {
```
Comment (MMortaga) [inline public/css/login.css]: Don't forget to add the `type` value in quotation marks like following:
`.login-card input[type='email'], input[type='password'] {`

File: public/js/modal.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+
+// When the user clicks on <span> (x), close the modal
+span.onclick = function () {
+    modal.style.display = "none";
+}
+
+// When the user clicks anywhere outside of the modal, close it
+window.onclick = function (event) {
+    if (event.target == modal) {
```
Comment (MMortaga) [inline public/js/modal.js]: Use triple equal here. Always try to satisfy the linter as much as you can.

File: public/js/projects.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    projectname: projectnametxt.value,
+    sector: sectortxt.value,
+    coordinationstatusid: statusselect.value,
+    location: locationtxt.value,
+  };
+  request('POST', '/getprojects', filterparams, cb);
+}
+
+function renderFitchedProjects(projects) {
+  projectstablebody.innerHTML = '';
```
Comment (MMortaga) [inline public/js/projects.js]: Use .textContent here and avoid .innerHTML unless it's extremely necessary and doesn't leave a security risk.

File: src/authentication/authentication.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
+const { sign, verify } = require('jsonwebtoken');
+require('env2')('config.env');
+
+const createCookie = (user, cb) => {
+  sign(user, process.env.SECRET, (signerror, token) => {
+    if (signerror) { cb(signerror); } else {
```
Comment (MMortaga) [inline src/authentication/authentication.js]: Fix your indentation.

File: src/controllers/error.js [hunk 5]
```diff
@@ -9,6 +9,7 @@ exports.client = (req, res) => {
 };
 
 exports.server = (err, req, res, next) => {
+  console.log('500err', err);
```
Comment (MMortaga) [inline src/controllers/error.js]: Handle errors properly and remove any console.logs

File: src/database/db_build.sql [hunk 6]
```diff
@@ -61,14 +62,34 @@ CREATE TABLE Projects (
...
 	description TEXT NOT NULL,
     FOREIGN KEY (coordination_status_id) REFERENCES coordination_status(id) ON DELETE CASCADE,
-    FOREIGN KEY (project_status_id) REFERENCES project_status(id) ON DELETE CASCADE
+    FOREIGN KEY (project_status_id) REFERENCES project_status(id) ON DELETE CASCADE,
+	FOREIGN KEY (sector_id) REFERENCES sectors(id) ON DELETE CASCADE
+
 );
 
-Insert INTO users (name, username, pass, email ,id_number ,mobile ,role ,job_title)VALUES ('Marwan Elkhoudary', 'melkhoudary', '$2a$10$b87/SM7GS1rjoFV2i8z5X.dGGQcZPW8l.fhVHxgg89GnOBAGLlf7a', 'marwangaza@hotmail.com', '800640864', '0595599633', 'admin', 'Head of Department');
+Insert INTO users (name, username, pass, email ,id_number ,mobile ,role ,job_title)VALUES ('Marwan Elkhoudary', 'melkhoudary', '$2a$10$b87/SM7GS1rjoFV2i8z5X.dGGQcZPW8l.fhVHxgg89GnOBAGLlf7a', 'marwangaza@hotmail.com', '800640864', '0595599633', 'admin', 'Head of Department'),
```
Comment (MMortaga) [inline src/database/db_build.sql]: Capitalize all your SQL keywords

File: src/database/queries/getcoordinationstatus.js [hunk 7]
```diff
@@ -0,0 +1,5 @@
+const dbconnection = require('../db_connection');
+
+const getCoordinationStatus = () => dbconnection.query('select * from coordination_status');
```
Comment (MMortaga) [inline src/database/queries/getcoordinationstatus.js]: Capitalize your queries.

File: src/authentication/authentication.js (hunk 4 above)
Comment (amd711) [reply src/authentication/authentication.js]: I cannot see any notifications by eslint regarding indentation at that file.

File: public/css/login.css (hunk 1 above)
Comment (amd711) [reply public/css/login.css]: done

File: public/js/modal.js (hunk 2 above)
Comment (amd711) [reply public/js/modal.js]: done

File: public/js/projects.js (hunk 3 above)
Comment (amd711) [reply public/js/projects.js]: done

File: src/controllers/error.js (hunk 5 above)
Comment (amd711) [reply src/controllers/error.js]: done

File: src/database/db_build.sql (hunk 6 above)
Comment (amd711) [reply src/database/db_build.sql]: Done, as well as query js files.

File: src/database/queries/getcoordinationstatus.js (hunk 7 above)
Comment (amd711) [reply src/database/queries/getcoordinationstatus.js]: done

**Pull request merged**